tests/test_preprocess.py

```diff
@@ -33,10 +32,11 @@ def test02Downsample(self):
         self.assertTrue(np.allclose(results, vols, atol=1e-7))
 
     def test03Vol2img(self):
-        results = np.load(os.path.join(DATA_DIR, 'clean70SRibosome_down8_imgs32.npy')).T # RCOPT
-        vol = np.load(os.path.join(DATA_DIR, 'clean70SRibosome_vol_down8.npy')).T # RCOPT
-        rots = qrand_rots(32, seed=0)
-        imgs_clean = vol2img(vol, rots)
+        results = np.load(os.path.join(DATA_DIR, 'clean70SRibosome_down8_imgs32.npy')).T
+        vols = np.load(os.path.join(DATA_DIR, 'clean70SRibosome_vol_down8.npy')).T
+        rots = np.load(os.path.join(DATA_DIR, 'rand_rot_matrices32.npy'))
+        rots = np.moveaxis(rots, 2, 0)
+        imgs_clean = vol2img(vols, rots)
```

Comment:
Since we're already replacing the binary blobs, can we replace `vol2img` by `Volume.project` (see #265) here? Otherwise we'll have two set of blobs littering up the repository.

Reply:
Changed to use Volume.project in commit b6890aa. Maybe it should be moved to unit tests for Volume class. The vol2img function was removed in commit  50091fd.

Reply:
I had read this too quickly. As Garrett pointed out during the meeting, this particular test is no longer needed once we remove `vol2img` (since it specifically tests that function). According to #265, the only other place it was used is in `test_covar2d.py`. Was that fixed already?

Reply:
The old way to generate consistent images via `vol2img` has been removed for unit tests and tutorial example of Cov2d.

Reply:
:+1: